packages/renderer/src/lib/kube/details/KubeContainerArtifact.svelte

```diff
@@ -2,7 +2,9 @@
 import type { V1Container } from '@kubernetes/client-node';
 
 import Cell from '/@/lib/details/DetailsCell.svelte';
-import KubeContainerPorts from '/@/lib/kube/details/KubeContainerPorts.svelte';
+import { WorkloadKind } from '/@api/kubernetes-port-forward-model';
+
+import KubePorts from './KubePortsList.svelte';
```

Comment:
Could we use the same name for the imported resource and the original one, `KubePortsList`?